app/view/webEngine/webEngineView.js

```diff
@@ -32,15 +32,57 @@
  */
 SDL.WebEngineView = Em.ContainerView.create({
       elementId: 'webEngineView',
+
       classNameBindings: [
         'SDL.States.webViewApp.active:active_state:inactive_state'
       ],
+
       childViews: [
-        'TemplateTitleLabel'
+        'TemplateTitleLabel',
+        'ExitButton',
+        'MenuButton'
       ],
 
+      isExitButtonVisible : true,
+
       TemplateTitleLabel : SDL.Label.extend({
         elementId: 'template_title_label',
         contentBinding: 'SDL.SDLController.model.templateConfiguration.template'
-      })
+      }),
+
+      ExitButton: SDL.Button.extend({
+        classNames: 'button ExitButton',
+        classNameBindings: ['parentView.isExitButtonVisible::inactive_state'],
+        text: 'EXIT',
+        commandID: -2,
+        target: 'SDL.SDLController',
+        action: 'onCommand',
+        onDown: false
+      }),
+
+      MenuButton: SDL.Button.extend({
+        classNames: 'button MenuButton',
+        classNameBindings: ['parentView.isExitButtonVisible:inactive_state'],
+        text: 'MENU',
+        target: 'SDL.SDLController',
+        action: 'openCommandsList',
+        onDown: false
+      }),
+
+      trackApplicationCommands: function() {
+        if (SDL.SDLController.model) {
+          let commands = SDL.SDLController.model.get('currentCommandsList');
```

Comment:
Maybe it is worth adding a check for `commands` validity?

Reply:
@YarikMamykin fixed in https://github.com/LuxoftSDL/sdl_hmi/commit/75629fd3622dc0114f9d2b31198f501d330a43a9